e2e/scenario/filters1.spec.js

```diff
@@ -0,0 +1,1521 @@
+/* eslint-disable no-undef */
+
+//TODO: getProperties from UIVeri5
+//var ui5ControlConverter = require("../../ui5ControlConverter");
+
+/*
+1) Test standard properties
+2) Test new properties adding from ui5 api
+3) Test with removing empty properties
+4) Testing with binding path/bindingContextPath
+4a) Testing with binding path/context on prec,next, parent element
+5) Testing with i18nText
+5a) Testing with i18nText on prec,next, parent element
+6) Testing with boolean, number properties
+7) Test with empty values, wrong property names, wrong metadata
+8) Test with index
+9) Test with aggregation bindings
+10) Test with ancestors
+11) Test with bindingpath aggregations
+12) Test with chaining of elements
+13) Test view Id & Name
+14) Test read UI Control and all properties immidietaly
+15) Test dom properties
+16) Test descentor/child properties
+17) Test assertions
+ */
+
+describe("filters1", function () {
+
+  it("step0v1: check Page has descendant text - wait for property only not element", async function () {
+    var ui5ControlProperties = {
+      "elementProperties":{"metadata":"sap.m.Page","mProperties":{
+        "id": "container-cart---welcomeView--page"}
+      },
+      "descendantProperties": {"metadata": "sap.m.Text", "mProperties": {
+        "text":[{"path": "i18n>welcomeCarouselShipping"}]
+      }}
+    };
+    await ui5.common.assertion.expectAttributeToBe(ui5ControlProperties, "showHeader", "true", 0, 1, 30000);
```

Comment:
note: we can declare the custom matchers in conf.js

Reply:
what do you mean custom matcher?